.github/PULL_REQUEST_TEMPLATE/major-version-up.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Major Version Up
+about: I'd like to increment the major version of a core plan
+labels: C-major-version-up
+
+---
+
+Hello - thanks for contributing to Habitat Core Plans!
+
+You've indicated that you're incrementing the major version of software packaged as a Habitat Core Plan. Please review and include the following information:
+
+- [ ] A link to the upstream Release notes or Changelog.
+
+If the Plan doesn't include a tests directory, please describe how we can verify that the resulting package is working as intended.  If this version includes new features, please include those in the description. Please read [RFC0004](https://github.com/habitat-sh/core-plans-rfcs/blob/master/_RFCs/0004-testing-pull-requests.md) for more details
+
+- [ ] Description of how to validate the package _**OR** addition of tests to validate the package.
```

Comment:
extra `_` ?

Reply:
Good catch. I didn't see those at all when editing. I'm going to go with a Vim plugin was being "helpful" 😂    Fix incoming